### PR TITLE
Apply CI/CD to project with Travis CI, only run test and deploy on branch master

### PR DESCRIPTION
**Pull request by nguyen315**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  - 12
+
+jobs:
+  include:
+    - name: server
+      before_script:
+        - cd $TRAVIS_BUILD_DIR/server
+        - npm install
+      script:
+        - echo "Hello Server" # will replace with test
```
Comment (sonns) [inline .travis.yml:12]: @nguyen315 
please help me change the content : 
 echo "Started server"

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          on:
+            branch: master
+          skip_cleanup: true
+
+    - name: client
+      before_script:
+        - cd $TRAVIS_BUILD_DIR/client
+        - npm install
+      script:
+        - echo "Hello Client" # will replace with test
```
Comment (sonns) [inline .travis.yml:28]: @nguyen315  same above

File: server/.gitignore [hunk 3]
```diff
@@ -2,6 +2,7 @@
 /dist
 /node_modules
 .env
+package-lock.json
```
Comment (sonns) [inline server/.gitignore:6]: refactor code (empty line)

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
+language: node_js
+node_js:
+  - 12
+
```
Comment (sonns) [inline .travis.yml:4]: @nguyen315 
Remove empty line

Comment (nguyen315) [reply .travis.yml:4]: Prettier format add new line at EOF, i think we should keep this format

**Pull request merged**